src/itest/kotlin/nl/lifely/zac/itest/ZaakRestServiceTest.kt

```diff
@@ -79,7 +80,9 @@ class ZaakRestServiceTest : BehaviorSpec({
                 groupName = TEST_GROUP_A_DESCRIPTION,
                 startDate = DATE_TIME_2020_01_01,
                 communicatiekanaal = COMMUNICATIEKANAAL_TEST_1,
-                vertrouwelijkheidaanduiding = DOCUMENT_VERTROUWELIJKHEIDS_AANDUIDING_OPENBAAR
+                vertrouwelijkheidaanduiding = DOCUMENT_VERTROUWELIJKHEIDS_AANDUIDING_OPENBAAR,
+                description = ZAAK_DESCRIPTION_2,
+                toelichting = ZAAK_EXPLANATION_1
             )
             Then("the response should be a 200 HTTP response with the created zaak") {
                 response.code shouldBe HTTP_STATUS_OK
```

Comment:
What are we going to do with this? Should I create a story to clean this up?

Reply:
We can also have the `https://github.com/lectra-tech/koson` in, if possible

Reply:
I don't understand you.. These values are defined globally in `ItestConfiguration`. Or what do you mean exactly?

Reply:
And what do you mean by header entries in this context?

This code just validates a custom and rather large JSON response body from one of our ZAC REST endpoints.

Reply:
Ah, you are talking about `HTTP_STATUS_OK` here?? Yeah, that's a little unfortunate. It's only because `okhttp3` does not expose these (as far as I know). They do offer e.g.:

``` 
val isSuccessful: Boolean
  get() = code in 200..299
```

but that's a bit too broad I find. I did not want to introduce yet another library just for these constants so I personally think it's just fine to define them as custom constants. It's just for integration tests anyway right?    

But improvements are always welcome of course. :-)